### PR TITLE
[airvisualnode] Change dependency group id to version available on Maven Central

### PR DESCRIPTION
**Pull request by kaikreuzer**
...and also included dependency in the binding.

Related to https://github.com/openhab/openhab-distro/issues/1256

Fixes sandbox release build https://ci.openhab.org/view/Sandbox/job/sandbox-openhab3-release/170/

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: bundles/org.openhab.binding.airvisualnode/pom.xml [hunk 1]
```diff
@@ -14,11 +14,17 @@
...
+      !jcifs
+    </bnd.importpackage>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.samba.jcifs</groupId>
       <artifactId>jcifs</artifactId>
-      <version>1.3.17</version>
+      <version>1.3.14-kohsuke-1</version>
```
Comment (J-N-K) [inline bundles/org.openhab.binding.airvisualnode/pom.xml:27]: Not 100% sure, but IIRC there was a reason to chose 1.3.17. Maybe a security issue?

Comment (kaikreuzer) [reply bundles/org.openhab.binding.airvisualnode/pom.xml:27]: My problem is that 1.3.17 does not seem to exist at all in any public repo...

Comment (J-N-K) [reply bundles/org.openhab.binding.airvisualnode/pom.xml:27]: https://mvnrepository.com/artifact/jcifs/jcifs/1.3.17

Would this work?

Comment (kaikreuzer) [reply bundles/org.openhab.binding.airvisualnode/pom.xml]: Thanks, that looks good indeed!

**Review by wborn — APPROVED** [review]

**Pull request merged**